app/config.py

```
@@ -0,0 +1,24 @@
+"""Configuration handling for the Content automation service."""
+
+from functools import lru_cache
+from typing import List, Optional, List
```

Ruff (0.13.1)

Redefinition of unused `List` from line 4

Remove definition: `List`

(F811)